release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -871,23 +889,27 @@ With this release, the logical volume manager (LVM) cluster custom resource (CR)
 
 [id="ocp-4-14-storage-gcp-filestore-csi-driver"]
 ==== GCP Filestore storage CSI Driver Operator is generally available
 {product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Compute Platform (GCP) Filestore Storage. The GCP Filestore CSI Driver Operator was introduced in {product-title} 4.12 with Technology Preview support. The GCP Filestore CSI Driver Operator is now generally available. For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file[Google Compute Platform Filestore CSI Driver Operator].
 
 [id="ocp-4-14-storage-automigration-vsphere"]
 ==== Automatic CSI migration for VMware vSphere
-This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Although storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+The Automatic CSI migration for VMware vSphere feature automatically translates in-tree objects to their counterpart CSI representations and, ideally, must be completely transparent to users. Although storage class referencing to the in-tree storage plug-in continues to work, consider switching the default storage class to the CSI storage class.
 
-In {product-title} 4.14, CSI migration for vSphere is enabled by default under all circumstances and requires no action by an administrator.
+In {product-title} {product-version}, CSI migration for vSphere is enabled by default under all circumstances and requires no action by an administrator.
 
-However, if you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 or 4.13 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3l or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. If you do not want to update vSphere, you can proceed with an {product-title} upgrade by performing an administrator acknowledgment. However, when using the administrator acknowledgment, known issues can occur. link:https://access.redhat.com/node/7011683[Before proceeding with the administrator acknowledgement, carefully read this knowledge base article].
+If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 or 4.13 to {product-version}, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. If you do not want to update vSphere, you can proceed with an {product-title} upgrade by performing an administrator acknowledgment. However, with using the administrator acknowledgment, known issues can occur. Before proceeding with the administrator acknowledgement, carefully read the link:link:https://access.redhat.com/node/7011683[Knowledge Base article].
```

Comment:
```suggestion
If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 or 4.13 to {product-version}, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. If you do not want to update vSphere, you can proceed with an {product-title} upgrade by performing an administrator acknowledgment. However, with using the administrator acknowledgment, known issues can occur. Before proceeding with the administrator acknowledgement, carefully read the link:https://access.redhat.com/node/7011683[Knowledge Base article].
```